tutorials/R/Geospatial-skills/intro-vector-r/05-vector-raster-integration-advanced/05-vector-raster-integration-advanced.Rmd

```diff
@@ -402,7 +402,7 @@ for example, plot locations that we are sampling on the ground.
 
 <figure>
     <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/spatialData/BufferSquare.png">
-    <img src="{{ site.baseulr }}/images/spatialData/BufferSquare.png"></a>
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/spatialData/BufferSquare.png" alt="Extraction of raster information using a polygon boundary."></a>
```

Comment:
Good catch! I always searched for {{ site.baseurl }} and didn't think about typos!